src/app.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// Write your code here
+import readline from 'readline';
+import { generateRandomNumber } from './modules/generateRandomNumber';
+import { checkIsValidUserInput } from './modules/checkIsValidUserInput';
+import { getBullsAndCows } from './modules/getBullsAndCows';
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const numberToGuess = generateRandomNumber();
+
+terminal.question('Guess the number', (userNumber) => {
+  if (!checkIsValidUserInput(userNumber)) {
+    return 'Input is not a valid 4-digit number';
+  }
+
+  if (userNumber === numberToGuess) {
```

Comment:
You should provide feedback to the user when they guess the number correctly, such as a congratulatory message.